src/api/generatedTypes.ts

```diff
@@ -595,6 +591,10 @@ export type CreateRejectedCorpusItemInput = {
 export type CreateScheduledCorpusItemInput = {
   /** The ID of the Approved Item that needs to be scheduled. */
   approvedItemExternalId: Scalars['ID'];
+  /** Free-text entered by the curator to give further detail to the manual schedule reason(s) provided. */
+  manualScheduleReasonComment?: InputMaybe<Scalars['String']>;
```

Comment:
Why did these two definitions move? I'm not understanding how this connects to the bug fix.

Reply:
I ran `npm run api:generate-types` and it re-generated the types. I believe these are the recent back-end changes done by Nina

Reply:
I can discard the changes to this file as its not relevant to the bug fix

Reply:
No problem, just wasn't understanding it!